### ISSUE TITLE
Device ok marking wrong at Status Page

### ISSUE THREAD

**Issue by ChaosEmpire** [Map-A-Droid/MAD#845]
Lets assume we set 3 min as time to mark a device as OK at Status Page
Next Action is displayed 5 min

The device hasnt send data for 3.5 min and we dont see a checkmark for OK

Correct is...the device is fine for Next Action time + configured warn time
So it should get marked as OK, until next action + warn time is passed 

At the moment devices in wait after teleport , appear to be broken

**Comment by JabLuszko** [comment]
This is not "device OK" marking, this is "Last update OK" marking. It was never designed to use Next Action, however I see your point here and it kinda make sense if you want this a "device OK" status.

Personally not gonna touch it, already had some talks about how useless this thing is and why somethingsomething there and why not somethingsomething else. If anyone wants to do it - sure :)

**Comment by Expl0dingBanana** [comment]
I do not believe this is an issue due to that checkbox exists only for determining if it has taken an action in the last X seconds.  I am open to discussing this but if the consensus is it does not align with the FR I would like to close this issue.
